### PR TITLE
Update SQL main select, JSON format update, remove all SQL dir

### PR DESCRIPTION
**Pull request by iandalrymple**
1) Updated the SQL main select statement with the correct number of joins and so on. 
2) JSON format file updated to include the ID's of each edge 
3) Removed the old SQL directories since they were dupes 
4) Added a code snippet file in the SQL directory that contains a JS function for parsing the results of a single cookie id select. This can be patched into the  crawler code to create the json string to be sent back. The file also contains the calling code that performs the query. 
 This is my first pull request in the non 362 style so let me know if I messed anything up

### CODE REVIEW

File: SQL/JSON_EXAMPLE.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	"MetaData": {
+		"CookieId": 4,
+		"User": "Ian",
+		"SearchType": "BFS",
+		"SearchDepth": 3,
+		"RootUrl": "google.com"
+	},
+	"Edges": [{
+		"SourceUrl": "google.com",
+		"SourceId": 1,
```
Comment (nigelrobbins3) [inline SQL/JSON_EXAMPLE.json:11]: :+1:

File: SQL/JSON_Parsing.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+			res.send(err);
+			return;
+		}
+		res.type('text/plain');	
+		res.send(QueryResultsToJson(JSON.stringify(rows)));
+	});
+}
+	
+	
+function CrawlResultsToJson(raw)
```
Comment (nigelrobbins3) [inline SQL/JSON_Parsing.js:32]: Typically javascript function names are camelCase, while class constructors are PascalCase. Javascript didn't have real classes until recently, so people used the casing to know when they should expect the function to return a class instance.

This is an internal project, so we can do what we like, but my functions will be camelCase.

**Review by nigelrobbins3 — APPROVED** [review]
Looks good. You don't have to fix the casing before you merge this, since the snippet is non-functional anyway. @aharcourt can decide how/if she wants to incorporate it.

File: SQL/JSON_Parsing.js (hunk 2 above)
Comment (iandalrymple) [reply SQL/JSON_Parsing.js:32]: Thats cool - I dont use JS ever day to day. Exclusively .NET so I dont know about the format idiosyncrasies. We probably should have laid out some rules before we started. But I will follow the rules you speak of going forwar

**Pull request merged**